packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -2537,6 +2537,24 @@ QUnit.module('Visibility of control elements', {
 });
 
 QUnit.module('Events', moduleConfig, () => {
+    QUnit.test('should output correct size value for panes in onResizeEnd event (T1240125)', function(assert) {
+        const resizeHandlerStub = sinon.stub().callsFake(function(eventArgs) {
+            // Access the items and their sizes from eventArgs if that's how they're passed
+            const newItems = eventArgs.component.option('items');
+            assert.notStrictEqual(newItems[0].size, oldItems, 'Size of the first item should be updated');
+        });
+
+        this.reinit({
+            onResizeEnd: resizeHandlerStub,
+            dataSource: [{ text: 'Pane_1' }, { text: 'Pane_2' }]
+        });
+        const oldItems = this.instance.option('items')[0].size;
```

Comment:
[naming]
Now this variable's name does not meet the value.
```suggestion
        const firstPaneInitialSize = this.instance.option('items')[0].size;
```

Reply:
[85ecf61](https://github.com/DevExpress/DevExtreme/pull/27793/commits/85ecf6149775d598cf2b711ea1ec5f599bb52b25)